util/EIP-1024/src/lib.rs

```diff
@@ -0,0 +1,139 @@
+extern crate saltbabe;
```

Comment:
extern crate is not needed for `rust2018`, replace with all with `use arrayref::array_ref`

Reply:
ok